Sources/PatchDay/Hormones/Details/HormoneDetailViewController.swift

```diff
@@ -44,7 +44,7 @@ class HormoneDetailViewController: UIViewController,
         setBackButton()
     }
 
-    override func viewWillAppear(_ animated: Bool) {
```

Comment:
note to future self: this is because all the other methods are this way and there is a warning in xcode about it.